src/domdiv/draw.py

```diff
@@ -1193,6 +1193,8 @@ def replace_image_tag(
             # Potion
             (r"(\d+)\s*\<\*POTION\*\>", "potion_small.png", 2, 140, 1.5),
             (r"Potion", "potion_small.png", 1.2, 100),
+            # Sun
+            (r"Sun", "sun.png", 1.2, 105),
```

Comment:
I added the rule to replace it in card text (though I'm not sure if that would impact anywhere else yet). There are a few other cards that have "sun" in their text...

Reply:
I wonder whether it shouldn't be `SunToken` or `RisingSun` instead of `Sun` to avoid accidental use?

Reply:
I checked a few cards with "sun" in their names and this seems to work OK, so approving. Keep an eye on it though.

Reply:
I'm fine with changing it to `SunToken` Let me do that....

Reply:
Done, changed to `SunToken`